docs/book/src/tasks/installation.md

```diff
@@ -1,173 +1,30 @@
 # Installation
```

Comment:
I don't know that we need to change it in this PR, but this file looks like it's basically the quickstart minus setting up the workload cluster. Should we get rid of this file?

Reply:
I'm opening an issue for this. I have to investigate if it is used somewhere else